### DISCUSSION TITLE
Kaboom v3000-beta3 does not unregister EventControllers when load a new screen

### DISCUSSION THREAD

**Discussion by kdetry** [replit/kaboom#702] [Q&A] [answered]
When we change the screen by using the go method, event handlers which are registered by using the onKeyPress method are still active. We need to cancel them manually by using the cancel method of the EventController structure.

I couldn't be sure if it was a bug

**Comment by slmjkdbtl** [comment, marked as answer]
Yep it's a bug, b37110d6936b69efd68e7d1d97938dfbbcf65fce should fix it
